helm/templates/consultingtypeservice/consultingtypeservice-deployment.yaml

```
@@ -0,0 +1,130 @@
+    matchLabels:
+      app: consultingtypeservice
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: consultingtypeservice
+    spec:
+      containers:
+        - name: consultingtypeservice
+          image:  {{ .Values.consultingTypeService.imageName }}:{{ .Values.consultingTypeService.imageVersion }}          
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)